src/main/app/case/case-api.test.ts

```diff
@@ -172,6 +172,68 @@ describe('CaseApi', () => {
     expect(userCase).toStrictEqual({ id: '1', state: State.Draft, divorceOrDissolution: serviceType });
   });
 
+  test('Should prioritise Submitted case', async () => {
+    const mockDraftCase = {
+      id: '1',
+      state: State.Draft,
+      case_data: { divorceOrDissolution: DivorceOrDissolution.DIVORCE },
+    };
+    const mockSubmittedCase = {
+      id: '1',
+      state: State.Submitted,
+      case_data: { divorceOrDissolution: DivorceOrDissolution.DIVORCE },
+    };
+    mockApiClient.findExistingUserCases.mockResolvedValue([mockDraftCase, mockSubmittedCase]);
+
+    const userCase = await api.getExistingUserCase(DivorceOrDissolution.DIVORCE);
+
+    expect(userCase).toStrictEqual({
+      id: '1',
+      state: State.Submitted,
+      divorceOrDissolution: DivorceOrDissolution.DIVORCE,
+    });
+  });
```

Comment:
why not just put mockSubmittedCase

Reply:
you can't because the format is returned differently - I just copied the format of the other tests